src/app/_components/MediaQuery.tsx

```diff
@@ -1,19 +1,17 @@
-'use client';
-
-import { css } from '@emotion/react';
+import { css } from '@kuma-ui/core';
 
 export function MediaQuery() {
-  return <p css={styles.text}>MediaQuery</p>;
+  return <p className={styles.text}>MediaQuery</p>;
 }
 
 const styles = {
   text: css`
     color: orange;
     font-size: 32px;
-    @media screen and (375px <= width < 800px) {
+    @media screen and (sm <= width < md) {
```

Comment:
Using kuma-ui, tokens which are defined on kuma.config.ts, like `sm` or `md`, can be used as breakpoint